docs/glossary.rst

```diff
@@ -0,0 +1,20 @@
+.. _kinto-glossary:
+
+Glossary
+########
+
+
+.. glossary::
+
+    Cliquet Protocol
+            A system of rules that explains the way to interact with Kinto,
+            and how its data is organized.
+
+    HTTP API
+            The definition of a set of HTTP request messages along with the
```

Comment:
I'm ok with that.

I wish we'd have *endpoint* like in Cliquet glossary (http://cliquet.readthedocs.org/en/latest/reference/glossary.html)